plugin.rb

```diff
@@ -99,6 +99,9 @@ def user_voted
     add_to_serializer(:topic_list_item, :vote_count) { object.vote_count }
     add_to_serializer(:topic_list_item, :can_vote) { object.can_vote? }
     add_to_serializer(:topic_list_item, :user_voted) { object.user_voted?(scope.user) if scope.user }
+    add_to_serializer(:topic_list_item, :include_vote_count?) { object.can_vote? }
```

Comment:
It would be best if you passed the third parameter `define_include_method = false` to the other `add_to_serializer` calls, from a few lines above. Otherwise, `include_` methods are basically defined twice.